solutions/usaco-940.mdx

```diff
@@ -87,6 +87,62 @@ public class Factory {
 ```
 
 </JavaSection>
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+
+signed main(){
+    // uncomment before submitting to usaco
+    // freopen("factory.in", "r", stdin);
+    // freopen("factory.out", "w", stdout);
+
+    //number of nodes
+    int n;cin>>n;
+    
+    //outgoing bool array, stores if we go out from this array or not,initialised with false
+    vector<bool> outgoing(n,false);
```

Comment:
spaces between operators please

Reply:
you can edit these changes if you would like @SansPapyrus683

Reply:
if i'm directly committing something it's going to be a whole remake